media-gfx/qimgv/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+	<email>jeff.no.zhao@gmail.com</email>
+	<name>Jiayi Zhao</name>
+</maintainer>
```

Comment:
Please also include proxy-maint team so we get CC-ed on bugs. Grep other packages or look at our wiki page to see how to do that.